cardano-cli/src/Cardano/CLI/Shelley/Key.hs

```diff
@@ -381,6 +383,11 @@ readVerificationKeyOrTextEnvFile asType verKeyOrFile =
     VerificationKeyFilePath (VerificationKeyFile fp) ->
       readKeyFileTextEnvelope (AsVerificationKey asType) fp
 
+data PaymentSource
```

Comment:
Let's move this to `cardano-cli/src/Cardano/CLI/Shelley/Parsers.hs`

Reply:
Unfortunately, I can't move it because `VerificationKeyTextOrFile` is the type of the constructor argument which is defined in this file.

Reply:
Unfortunately I can't move it to `Parsers` because `AddressCmd` definition in `Commands` needs it.  Moving the type into `Parsers` would create a cyclic import dependency between `Parsers` and `Commands`.